.github/ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,11 @@
 
 **What is the [`ember-cli-rails-addon`][addon] version (from `package.json`)?**
 
+**What is your application server (e.g. `puma`, `unicorn`, `thin`, `webrick`,
+etc.)?**
+
+**Is your application server multi-threaded or multi-process?**
```

Comment:
This phrasing makes me think that I have to pick exactly one of these two options. Is that correct? Do we need to know this separately from the above question?